src/components/reservations/CustomerReservationRow.tsx

```diff
@@ -194,37 +195,65 @@ const CustomerReservationRow = ({ customer, reservation }: IProps): JSX.Element
           </div>
         </div>
       </div>
-      {firstInQueue && (
+      {(firstInQueue || !isCanceled) && (
```

Comment:
This check is here because we don't want to render the "buttons" <div> on the next line if its content is empty. So this check is only used here for rendering purposes and not for business logic.

Reply:
After a discussion, this will be changed to only `!isCanceled`